docker/Makefile

```diff
@@ -12,6 +12,18 @@ PUPPERWARE_ANALYTICS_STREAM ?= dev
 
 ifeq ($(IS_RELEASE),true)
 	VERSION ?= $(shell echo $(git_describe) | sed 's/-.*//')
+	# to work around failures that occur between when the repo is tagged and when the package
+	# is actually shipped, see if this version exists in dujour
+	PUBLISHED_VERSION ?= $(shell curl --silent 'https://updates.puppetlabs.com/?product=puppetserver&version=$(VERSION)' | jq '."version"' | tr -d '"')
```

Comment:
If we turn the `product` bit into a variable, it will be easier to copy / paste this to other projects I think

Reply:
updated